app/assets/src/components/views/phylo_tree/PhyloTreeVis.jsx

```diff
@@ -20,7 +20,7 @@ class PhyloTreeVis extends React.Component {
 
     this.handleNodeHover = this.handleNodeHover.bind(this);
 
-    this.sampleFields = SAMPLE_FIELDS;
+    this.sampleFields = SAMPLE_FIELDS.slice();
     this.sampleFields.splice(2, 0, {
```

Comment:
Whe not include this field in SAMPLE_FIELDS.
You can define a map of custom parsers for each field keyed by the name of the field or by a "type" parameter that you can include in SAMPLE_FIELDS.

If there is a parser defined for a field or a type you applied before displaying,